teuthology/task/install.py

```diff
@@ -520,6 +533,50 @@ def verify_package_version(ctx, config, remote):
         )
 
 
+def _install_rh_build(ctx, remote, config):
+    """
+    Installs RH build using ceph-deploy.
+
+    TODO: split this into at least two functions.
+
+    :param ctx: the argparse.Namespace object
+    :param rpm: list of packages names to install
+    :param config: the config dict
+    """
+    pkgs=['ceph-deploy']
+    for pkg in pkgs:
+      log.info("Check if ceph-deploy is already installed on system")
+      r= remote.run(
+          args = ['yum', 'list', 'installed',
+                 run.Raw(pkg)],
+          stdout=StringIO(),
+          check_status=False,
+      )
+      if r.stdout.getvalue().find(pkg) == -1:
+          log.info("Install ceph-deploy")
+          remote.run(args = ['sudo', 'yum', 'install', pkg, '-y'])
+      else:
+          log.info("Remove and Install ceph-deploy")
+          remote.run(args = ['sudo', 'yum', 'remove', pkg, '-y'])
```

Comment:
We shouldn't hit this if systems are in good state, but any bad runs its helpful the install works next time we run time.